docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc

```diff
@@ -821,6 +821,16 @@ Where `xx` is the switch port number
    mac-authentication enable-dynamic-vlan
    voice-vlan 100
 
+Radius CLI Login
+++++++++++++++++
+
+If you want to use the server PacketFence to authenticate users on the Brocade switch.
+
+ * Configure the radius server to send user authentication request to PacketFence
+   
+   aaa authentication login default radius local
+
+NOTE: Make sure to have a local account is case the switch can not reach the PacketFence server
```

Comment:
Make sure to have a local account in case the switch can not reached by the PacketFence server